classes/class3/cat-app/views/layouts/remove.handlebars

```diff
@@ -0,0 +1,7 @@
+<h1>Remove Cat</h1>
```

Comment:
I am guessing this was a mistake that you accidentally put a copy of this file here